File: 063_Nim/fizzbuzz.nim
# This is a comment.

for i in countup(1, 100):
    if ((i mod 3 == 0) and (i mod 5) == 0):
        echo("FizzBuzz")
    elif (i mod 3 == 0):
        echo("Fizz")
    elif (i mod 5 == 0):
        echo("Buzz")
    else:
        echo(i)

File: 049_Elixir/fizzbuzz.exs
# This is a comment.

defmodule FizzBuzz do

    def getValue(i) do
        cond do
            (Integer.mod(i, 3) == 0 && Integer.mod(i, 5) == 0) -> "FizzBuzz"
            Integer.mod(i, 3)  == 0 -> "Fizz"
            Integer.mod(i, 5)  == 0 -> "Buzz"
            :else -> i
        end
    end

end

for i <- 1..100 do
    IO.puts FizzBuzz.getValue(i)
end

